flake8_quotes/__init__.py

```diff
@@ -10,42 +10,40 @@ class QuoteChecker(object):
     name = __name__
     version = __version__
 
-    INLINE_QUOTES = {
-        # When user wants only single quotes
-        '\'': {
-            'good_single': '\'',
-            'good_multiline': '\'\'\'',
-            'bad_single': '"',
-            'bad_multiline': '"""',
-        },
-        '"': {
-            'good_single': '"',
-            'good_multiline': '"""',
-            'bad_single': '\'',
-            'bad_multiline': '\'\'\'',
-        },
-    }
+    INLINE_QUOTES = set(['"', "'"])
+    MULTILINE_QUOTES = set(['"""', "'''"])
 
     def __init__(self, tree, filename='(none)', builtins=None):
         self.filename = filename
 
     @classmethod
     def add_options(cls, parser):
+        # We don't care how it is exactly sorted, just that it is deterministic
```

Comment:
But honestly, why do we need a `set`? Keeping them as lists earlier on would also work and be deterministic.